runtime/oti/cfreader.h

```diff
@@ -75,6 +75,9 @@
 #define	CFR_STACKMAP_TYPE_BYTE_ARRAY			0x0E
 #define	CFR_STACKMAP_TYPE_CHAR_ARRAY			0x0F
 #define	CFR_STACKMAP_TYPE_BOOL_ARRAY			0x10
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define CFR_STACKMAP_TYPE_PRIMITIVE_OBJECT		0x11
+#endif /* #if defined(J9VM_OPT_VALHALLA_NEW_FACTORY_METHOD) */
```

Comment:
The comment is incorrect, Should be `/* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */`